src/viewer.js

```diff
@@ -294,6 +294,9 @@ $.Viewer = function( options ) {
 
     this.container.insertBefore( this.canvas, this.container.firstChild );
     this.element.appendChild( this.container );
+    this._createdElements = [];
```

Comment:
You don't seem to be reading this variable at all...

Reply:
@Vinith1919 you've resolved the comment but did not fix anything. Why keeping array of items you never access? Use it or delete it.